packages/enzyme-adapter-react-14/src/ReactFourteenAdapter.js

```diff
@@ -122,7 +130,21 @@ class ReactFourteenAdapter extends EnzymeAdapter {
         instance = null;
       },
       getNode() {
-        return instance ? instanceToTree(instance._reactInternalInstance).rendered : null;
+        if (!instance) {
+          return instance;
+        }
+        const tree = instanceToTree(instance._reactInternalInstance);
+        const rootFinder = findElement(tree, child => child.type === RootFinder);
+
+        if (!rootFinder) {
+          if (typeof options.wrappingComponent === 'function') {
+            throw new Error('`wrappingComponent` must render its children!');
+          } else {
+            throw new Error('Enzyme Internal Error: could not find <RootFinder />.');
+          }
+        }
+
+        return rootFinder.rendered;
```

Comment:
should we assert that `rootFinder.rendered` is a single react element matching the type of that passed to the wrapper?

Reply:
I don't think we need to. Because _we_ are the ones passing the root node as children to `<RootFinder />`, we can be confident that its children are correct. What `wrappingComponent` gets passed as `children` is actually `<RootFinder />`.

Now it _would_ be possible for the user to use `cloneElement()` or something to mess with the root by modifying `children.props.children` in their `wrappingComponent`, but I think, given that the user is highly unlikely to do that by accident, we don't need to go about making assertions for that case.

Does that seem reasonable?